collect_app/src/main/java/org/odk/collect/android/external/FormUriActivity.kt

```diff
@@ -49,6 +50,7 @@ class FormUriActivity : ComponentActivity() {
             logAnalytics(uriProjectId)
 
             if (projectId == currentProjectProvider.getCurrentProject().uuid) {
+                Timber.w("Form started via VIEW/EDIT actions")
```

Comment:
Are we not already logging this [through an analytics event](https://github.com/getodk/collect/blob/55c00b8ce6f2a08d94a84713362f56be4ad0a3ce/collect_app/src/main/java/org/odk/collect/android/external/FormUriActivity.kt#L49)? @lognaturel pointed that out to me the other day.

Reply:
Hmm let me check it...

Reply:
Right that event should be visible so we do not need a new one. Fixed.